README.md

```diff
@@ -828,11 +846,53 @@ axios.post('http://something.com/', params.toString());
 
 You can also use the [`qs`](https://github.com/ljharb/qs) library.
 
-###### NOTE
-The `qs` library is preferable if you need to stringify nested objects, as the `querystring` method has known issues with that use case (https://github.com/nodejs/node-v0.x-archive/issues/1665).
+> NOTE: 
+> The `qs` library is preferable if you need to stringify nested objects, as the `querystring` method has [known issues](https://github.com/nodejs/node-v0.x-archive/issues/1665) with that use case.
 
 #### Form data
 
+##### 🆕 Automatic serialization
+
+Starting from `v0.26.0`, Axios supports automatic object serialization to a FormData object if the request `Content-Type` 
+header is set to `multipart/form-data`.
+
+The following request will submit the data object as a form (Browser & Node.js):
+
+```js
+import axios from 'axios';
+
+axios.post('https://postman-echo.com/post', {x: 1}, {
+  headers: {
+    'Content-Type': 'multipart/form-data'
+  }
+}).then(({data})=> console.log(data));
+```
+
+In the `node.js` build, the ([`form-data`](https://github.com/form-data/form-data)) polyfill is used by default.
+
+You can overload the FormData class by setting the `env.FormData` config variable,
+but you probably won't need it in most cases:
+
+```js
+const axios= require('axios');
+var FormData = require('form-data');
+
+// OR
+// axios.defaults.env.FormData= FormData;
+
+axios.post('https://postman-echo.com/post', {x: 1, buf: new Buffer(10)}, {
+  headers: {
+    'Content-Type': 'multipart/form-data'
+  },
+
+  env: {  // OR
+    FormData
+  }
+}).then(({data})=> console.log(data));
+```
+
+##### Manual FormData passing
```

Comment:
I think for the time being we should start with this section, then if form-data is well accepted, we can switch and recommend the automatic serialization as the default method of choice.

Reply:
It seems we can be pretty sure that automatic serialization does at least the same thing as manually constructing FormData. Also, there is no deprecation sign or anything like that to confuse the user. Of course, the order can be changed, but I don't think it matters much.